model/riscv_insts_vext_arith.sail

```diff
@@ -42,6 +42,8 @@ function clause execute(VVTYPE(funct6, vm, vs2, vs1, vd)) = {
   let VLEN_pow = get_vlen_pow();
   let num_elem = get_num_elem(LMUL_pow, SEW);
 
+  if ~(valid_rd_mask(vd, vm)) then {handle_illegal(); return RETIRE_FAIL};
```

Comment:
First line of this file and the formatting is wrong, with 64 copies of this incorrect sequence in total.

Reply:
This issue is fixed now.

Reply:
> First line of this file and the formatting is wrong, with 64 copies of this incorrect sequence in total.

This file has large amounts of code duplication.  Can we use an execute function that provides a generic prologue and epilogue, implementing only the specific logic of each instruction differently?